From 3349d8a48ad1b957859a16643c912c4febec73b0 Mon Sep 17 00:00:00 2001
From: Pino Toscano <pino@kde.org>
Date: Sat, 4 Jun 2016 08:56:46 +0200
Subject: [PATCH] include <QStringList> instead of fwd declarations

slotAddKey needs the full QStringList definition, a forward declaration
is not enough
---
 model/kgpgsearchresultmodel.h | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/model/kgpgsearchresultmodel.h b/model/kgpgsearchresultmodel.h
index d754b12..c1d327e 100644
--- a/model/kgpgsearchresultmodel.h
+++ b/model/kgpgsearchresultmodel.h
@@ -20,12 +20,10 @@
 #define KGPGSEARCHRESULTMODEL_H
 
 #include <QAbstractItemModel>
+#include <QStringList>
 
 #include <kgpgcompiler.h>
 
-class QString;
-class QStringList;
-
 class KGpgSearchResultModelPrivate;
 
 /**
-- 
2.8.1

